tools/querytee/proxy_endpoint.go

```diff
@@ -224,6 +224,8 @@ func (p *ProxyEndpoint) executeBackendRequests(req *http.Request, resCh chan *ba
 			)
 		}
 
+		relativeDuration := expectedResponse.elapsedTime - actualResponse.elapsedTime
+		p.metrics.relativeDuration.WithLabelValues(req.Method, p.routeName).Observe(relativeDuration.Seconds())
```

Comment:
It might be worth including the comparison result as a label on this metric - I could imagine these results being skewed if one backend is quickly returning a lot of errors, for example.

Reply:
Great idea, will do